build/build.c

```diff
@@ -172,9 +174,113 @@ rpmRC doScript(rpmSpec spec, rpmBuildFlags what, const char *name,
     return rc;
 }
 
-static rpmRC buildSpec(BTA_t buildArgs, rpmSpec spec, int what)
+static rpmRC doBuildRequires(rpmSpec spec, int test) {
```

Comment:
In case of functions, the opening curly brace belongs to a line of its own. This is a recurring theme, please check and fix all newly introduced functions.